js/build/index.html

```diff
@@ -155,5 +170,5 @@
   To begin the development, run `npm start` or `yarn start`.
   To create a production bundle, use `npm run build` or `yarn build`.
 -->
-<script type="text/javascript" src="/react_paragraphs.vendor.min.js"></script><script type="text/javascript" src="/react_paragraphs.app.min.js"></script></body>
+<script type="text/javascript" src="/react_paragraphs.vendor.min.js"></script><script type="text/javascript" src="/react_paragraphs.app.min.js"></script><script type="text/javascript" src="/react_paragraphs.../../css/react_paragraphs.field_widget.min.js"></script><script type="text/javascript" src="/react_paragraphs.../../css/react_paragraphs.field_formatter.min.js"></script></body>
```

Comment:
URLs for these need some love.

Reply:
maybe we should just ignore the `build.index.html` file.

Reply:
Perhaps ignore everything in build?

Reply:
except the compiled js files we need?